code/modules/power/powernet.dm

```diff
@@ -84,18 +82,14 @@
 		for(var/obj/machinery/power/smes/S in nodes) // find the SMESes in the network
 			S.restore() // and restore some of the power that was used
 
-	// update power consoles
-	viewavail = round(0.8 * viewavail + 0.2 * avail)
-	viewload = round(0.8 * viewload + 0.2 * load)
-
 	// reset the powernet
 	load = delayedload
 	delayedload = 0
 	avail = newavail
 	newavail = 0
 
 /datum/powernet/proc/get_electrocute_damage()
-	if(avail >= 1000)
-		return clamp(20 + round(avail/25000), 20, 195) + rand(-5,5)
+	if(energy_to_power(avail) >= 1000)
+		return clamp(20 + round(energy_to_power(avail)/25000), 20, 195) + rand(-5,5)
```

Comment:
What exactly is 25000? it looks like watts but maybe it should have its own define or at least a comment explaining what this magic number is.

Reply:
Watts per damage, so WATTS would be the closest we have. I'll turn it into a define when I have the time.

Reply:
Actually it would make more sense if it was joules, and we just assume 1 second of contact, since this exact same code is used somewhere else for cells. I could turn this into a macro.

Reply:
Added defines and a macro for it.